docs/source/models/gbd2017/causes/causes.rst

```diff
@@ -23,11 +23,11 @@ causes of death or disease are mutually exclusive and collectively exhaustive.
 
     - ... down to the most detailed level
```

Comment:
Would be nice to represent this as a tree, such that you could see branches diverging